src/main/java/org/com/itpple/spot/server/service/impl/UserInfoServiceImpl.java

```diff
@@ -74,4 +74,30 @@ public void validateNickname(String nickname) {
             throw new NicknameValidationException();
         }
     }
+
+    public UserInfoResponse getUserInfo(Long userId) {
+        User user = userRepository.findById(userId)
+                .orElseThrow(() -> new UserIdNotFoundException("PK = " + userId));
+        return UserInfoResponse.from(user);
+    }
+
+    @Transactional
+    public UserInfoResponse updateUserInfo(Long userId, UserInfoRequest userInfoRequest) {
+        User user = userRepository.findById(userId)
+                .orElseThrow(() -> new UserIdNotFoundException("PK = " + userId));
+        if (userInfoRequest.nickname() != null && !userInfoRequest.nickname().isEmpty())
```

Comment:
```suggestion
        if (!StringUtils.isBlank(userInfoRequest.nickname()))
```
으로 하면 좀 더 읽기 편할 거 같아요!